src/timeline/timeline.style.tsx

```diff
@@ -157,3 +168,8 @@ export const TimelinePill = styled.div<TimelineStatusProps>`
         `;
     }}
 `;
+
+export const ErrorIcon = styled(ExclamationCircleFillIcon)`
```

Comment:
It's parent element `CircleIndicator` is of the same height and width already. Is there a reason why we still need this explicit style?

Reply:
It looks smaller without the explicit styling. 
<img width="464" alt="image" src="https://github.com/LifeSG/react-design-system/assets/58726408/3f31ad26-38ff-4030-96ac-a5643d19b714">

Reply:
Ah I know why now. the icon comes with the outer circle too. Ok then i guess we dont have a choice. We'll accept this implementation.

Reply:
On that note, it actually looks smaller still. If you look closely, the icon has a bounding box around it making it not `1.5rem` visually.

Reply:
Yeah you're right! It actually does look tiny bit smaller. Have bumped up the `CircleIndicator` to `1.8rem` to have the svg path size of 24px x 24px.